### PR TITLE
auction.co.kr - Social Tracker Copyright-notice maintenance

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [X] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://browse.auction.co.kr/search?keyword=%EB%85%B8%ED%8A%B8%EB%B6%81
 - https://browse.auction.co.kr/list?category=67040400
 - http://itempage3.auction.co.kr/DetailView.aspx?itemno=C764152631
 - http://itempage3.auction.co.kr/DetailView.aspx?itemno=C463404157
 - http://mitem.auction.co.kr/vip?itemNo=C739248252
 - https://m.auction.co.kr/
 - https://github.com/AdguardTeam/AdguardFilters/issues/14146

### Add your comment and screenshots

 - https://github.com/AdguardTeam/AdguardFilters/issues/14146 - The rules that will be deleted allow advertisement of Gmarket Inc. [^1][^2]

[^1]: https://ad.esmplus.com/ADS/SignIn/LogOn
[^2]: https://doc.gmarket.co.kr/esm/ad/da/%EC%A7%80%EB%A7%88%EC%BC%93_%ED%86%B5%ED%95%A9%EC%BB%A4%EB%A8%B8%EC%8A%A4%EA%B4%91%EA%B3%A0_%EC%83%81%ED%92%88%EC%86%8C%EA%B0%9C%EC%84%9C.pdf

- https://www.similarweb.com/website/auction.co.kr/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by piquark6046** [comment]
[지마켓_통합커머스광고_상품소개서.pdf](https://github.com/AdguardTeam/AdguardFilters/files/9989666/_._.pdf)

File: SpywareFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -20,6 +20,8 @@ imdb.com#%#//scriptlet("set-constant", "doWithAds", "noopFunc")
 ! https://developer.chrome.com/docs/privacy-sandbox/topics/
 ! TODO: replace with a scriptlet later: https://github.com/AdguardTeam/Scriptlets/issues/214
 #%#//scriptlet('set-constant', 'Document.prototype.browsingTopics', 'undefined')
+! https://github.com/AdguardTeam/AdguardFilters/pull/134619
+auction.co.kr#%#AG_onLoad(() => { Array.from(document.querySelectorAll('a[href*="through.auction.co.kr/common/SafeRedirect.aspx"]')).forEach((e) => { e.href = new URLSearchParams(e.href).get('next') }) })
```
Comment (Alex-302) [inline SpywareFilter/sections/general_extensions.txt:24]: What the purpose of this rule?

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt [hunk 2]
```diff
@@ -9,6 +9,7 @@
 !
 ! SECTION: Mobile apps - Regular rules
 !
+auction.co.kr##[class*="app-install"]
```
[the remark this answers is not in the document]
Comment (piquark6046) [reply AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:12]: My rule blocks their app banners on [mitem.auction.co.kr](http://mitem.auction.co.kr/vip?itemNo=C463404157) and [m.auction.co.kr](https://m.auction.co.kr/) (`a.link__app-install.sprite__footer--before.sprite__footer--after`).

[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:12]: Hm. Ok then.

File: SpywareFilter/sections/general_extensions.txt (hunk 1 above)
Comment (Alex-302) [reply SpywareFilter/sections/general_extensions.txt:24]: What do they collect here? I see url even without redundant parameters.

Comment (piquark6046) [reply SpywareFilter/sections/general_extensions.txt:24]: Hmm...Okay. I will delete that.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**